package.json

```diff
@@ -6,16 +6,19 @@
   "scripts": {
     "clean": "rm -rf dist",
     "start": "node dist/main.js -p 8080 --log-level=debug",
-    "serve_local": "npm run build && nodemon dist/main.js -- --dev --log-level=silly --use-cache",
+    "serve_local": "npm run build && nodemon dist/main.js -- --dev --log-level=silly --port=8040",
     "test": "NODE_ENV=test mocha --exit --reporter spec 'src/*_test.js' --require @babel/register --require test_setup.js",
     "deploy": "./deploy.sh",
     "deploy:docker": "DOCKER=1 ./deploy.sh",
     "set_default": "./set_default.sh",
     "lint": "flow && eslint --config .eslintrc ./*.js src",
     "prettyquick": "pretty-quick",
-    "build:noclean": "babel src --out-dir dist --source-maps --ignore src/testdata",
+    "build:noclean": "babel src --out-dir dist --source-maps --ignore src/testdata --config-file ./babel.config.js",
     "build": "npm run clean && npm run build:noclean",
-    "postinstall": "npm run build"
+    "postinstall": "npm run build",
+    "check:file": "bash -c 'git diff \"${@:1}\" --shortstat -- $0'",
```

Comment:
This gets the diff stats for a single file. The `${@:1` allows us to insert additional flags, like `--cached` when we call this script.